resources/profiles/Zonestar.ini

```diff
@@ -1,108 +1,129 @@
 # Zonestar print profiles
-
 # based on https://github.com/slic3r/slic3r-profiles/pull/19 by @ZONESTAR3D
 
 [vendor]
 # Vendor name will be shown by the Config Wizard.
 name = Zonestar
 # Configuration version of this file. Config file will only be installed, if the config_version differs.
 # This means, the server may force the PrusaSlicer configuration to be downgraded.
-config_version = 1.0.0
+config_version = 1.3.1
 # Where to get the updates from?
-config_update_url = https://files.prusa3d.com/wp-content/uploads/repository/PrusaSlicer-settings-master/live/Zonestar/
+config_update_url = https://github.com/ZONESTAR3D/Slicing-Guide/tree/master/PrusaSlicer/Profiles
 
 # The printer models will be shown by the Configuration Wizard in this order,
 # also the first model installed & the first nozzle installed will be activated after install.
 # Printer model name will be shown by the installation wizard.
 
-[printer_model:Z5]
-name = Zonestar Z5
-variants = 0.4
+[printer_model:Z8]
+name = Zonestar Z8
+variants = 0.4; 0.3; 0.6; 0.8; 1.0; 1.2
 technology = FFF
-bed_model = bed_220.stl
-bed_texture = 220_texture.svg
+bed_model = bed_310.stl
+bed_texture = 310_texture.svg
 default_materials = Generic PLA @Zonestar; Generic ABS @Zonestar; Generic PETG @Zonestar; Generic TPU @Zonestar
+thumbnail = Z8_thumbnail.png 
```

Comment:
You need to use different filename, otherwise old thumbnails will be not replaced. You can use for example Z8_thumbnail_v2.png (both here and actual filename). This applies for all previously existing Zonestar models (the models, where you are now updating thumbnails).